tinygrad/codegen/kernel.py

```diff
@@ -95,8 +95,11 @@ def __init__(self, ast:LazyOp, opts:Optional[LinearizerOptions]=None):
     self.group_for_reduce: List[int] = []
     self.upcasted: int = 0
     self.local_dims: int = 0
+    self.simd_local_dims: int = 0
```

Comment:
I really don't want to add more options here. Why do these have to be special?

Reply:
We could have TC infer in linearizer. 🤔

Reply:
This might not actually solve anything, but would it be better if this were just read from the tensor_cores? Tensor_cores is the only simd we have, I guess.

I need to think about other solutions. The thought here is: either we want to force the simd dimensions to stay in place, or we don't. If we do, then we might as well make it explicit. If we don't, then we need to check that the right conditions exist in linearizer. Many tradeoffs.

Let me know if you have some good idea.